### ISSUE TITLE
Bug in Socketworker Demo

### ISSUE THREAD

**Issue by iamtrask** [OpenMined/Grid#222]
It looks like there's a bug in dependencies. When I run "pip install socketio" it works but it breaks some other things because socketio requires an old version of setuptools.

```---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-3-faf5475d3c9a> in <module>()
----> 1 worker = gr.WebsocketGridClient(hook, "https://websocketgridtest.herokuapp.com/",id="grid")
      2 worker.connect()

/Users/atrask/anaconda/lib/python3.6/site-packages/grid-0.1.0a1-py3.6.egg/grid/websocket_client.py in __init__(self, hook, addr, id, is_client_worker, log_msgs, verbose, data)
     42 
     43         # Creates the connection with the server
---> 44         self.sio = socketio.Client()
     45         super().__init__(hook, id, data, is_client_worker, log_msgs, verbose)
     46 

AttributeError: module 'socketio' has no attribute 'Client'```

**Comment by mari-linhares** [comment]
@IonesioJunior

**Comment by IonesioJunior** [comment]
I'm on it.

**Comment by mari-linhares** [comment]
Fixed by #231
